Only partial re-export of `cookie` types?

So I noticed that there are some `pub use cookie::<Whatever>`s in tower-cookies, but it only covers some of the types. Like if you try to do
```rust
use tower_cookies::{Cookie, Cookies};

Cookie::build(...)
    .same_site(SameSite::Lax)
```
you are told
```
error[E0433]: failed to resolve: use of undeclared type `SameSite`
   --> path/to/file.rs:103:24
    |
103 |             .same_site(SameSite::Lax)
    |                        ^^^^^^^^ use of undeclared type `SameSite`
```
and there's no `SameSite` to import from `tower_cookies`. Obviously I could just add a dependency on `cookie` and import it from there, but I feel like this library would be less confusing if it either exported all of `cookie` (or at least the parts that are part of the cookie creation API) or none of `cookie`; exporting only parts makes the API surface only partially accessible "by default."

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -61,6 +61,9 @@ pub use cookie::Key;
 
 pub use cookie::Cookie;
 
+#[doc(inline)]
+pub use cookie;
+
 #[cfg(feature = "axum-core")]
 #[cfg_attr(docsrs, doc(cfg(feature = "axum-core")))]
 mod extract;

